benchmarks/unconstrained/default/cutest_benchmarks.jl

```diff
@@ -0,0 +1,71 @@
+##########################################################################
+#
+# Benchmark optimization algorithms by tracking:
+#
+# * Number of iterations
+# * Number of f_calls
+# * Number of g_calls
+# * Memory requirements (TODO)
+#
+####
+# unconstrained
+options = OptimizationOptions()
+cutest_problems = CUTEst.select(max_var=5,contype = :unc, custom_filter=x->x["derivative_order"]>=2)
+n = length(default_solvers)
+m = length(cutest_problems)
+f = open(join([version_dir, "cutest_benchmark.csv"], "/"), "w")
+write(f, join(["Problem", "Optimizer", "Converged", "Time", "Minimum", "Iterations", "f_calls", "g_calls", "f_hat", "f_error", "x_error"], ","))
+write(f, "\n")
+@showprogress 1 "Benchmarking..." for prob in cutest_problems
+    @show prob
+    output = []
+
+    nlp = CUTEstModel(prob)
+    d, constraints = optim_problem(nlp)
+    x0 = initial_x(nlp)
+
+    x_hat = copy(x0)
+    f_hat = obj(nlp, x_hat)
+    xs = []
+    times = []
+        for i = 1:n
+            if !(default_solvers[i] in (Newton(), NewtonTrustRegion()))
```

Comment:
Why do you not run these tests with the Newton solvers?

It worked well for me to remove this if statement, and use the following code to get results and mintime:

```jl
result = optimize(d, x0, default_solvers[i], options)
mintime = @elapsed optimize(d, x0, default_solvers[i], options)
```